src/debuggee.ts

```diff
@@ -75,9 +79,9 @@ export class Debuggee {
    *     that the user has a way of noticing.
    *     TODO(ofrobots): has this been renamed to `status` in the API?
    */
-  constructor(properties: DebuggeeProperties) {
+  constructor(properties: DebuggeeProperties, appInfo: AppInfo) {
```

Comment:
Can we set `properties.agentVersion` before calling this constructor, so we don't have to pass `appInfo` in as an argument?

Reply:
Done